src/Functions/dateDiff.cpp

```diff
@@ -193,18 +260,20 @@ class DateDiffImpl
 
 
 /** dateDiff('unit', t1, t2, [timezone])
-  * t1 and t2 can be Date or DateTime
+  * age('unit', t1, t2, [timezone])
+  * t1 and t2 can be Date, Date32, DateTime or DateTime64
   *
   * If timezone is specified, it applied to both arguments.
   * If not, timezones from datatypes t1 and t2 are used.
   * If that timezones are not the same, the result is unspecified.
   *
   * Timezone matters because days can have different length.
   */
+template <bool is_date_diff>
```

Comment:
How about template parameter "is_absolute" or "is_relative"?

Reply:
Done. Replaced it with "is_relative"